### PR TITLE
Adding Router Tree Skelton

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

**Comment by vanessayuenn** [comment]
@sumitarora this can be shipped without the actual router tree implementation right?

**Comment by sumitarora** [comment]
yes we just need to hide the div which displays router data

File: src/backend/adapters/angular2.ts [hunk 1]
```diff
@@ -78,6 +79,18 @@ export class Angular2Adapter extends BaseAdapter {
...
 
+  showAppRoutes(): void {
+    const root = this._findRoot();
+    try {
+      const routes = ParseRouter.parseRoutes(
+        ng.probe(root).componentInstance.router.root.registry);
+
+      this.showRoutes(routes);
+    } catch (error) {
+      console.log(error);
```
Comment (vanessayuenn) [inline src/backend/adapters/angular2.ts]: what error is being handled here?

Comment (sumitarora) [inline src/backend/adapters/angular2.ts]: Any parsing errors or inconsistent data due to API changes. Don't want users to get blocked.

Comment (vanessayuenn) [inline src/backend/adapters/angular2.ts]: :+1:

**Comment by vanessayuenn** [comment]
:100: :ship:

**Pull request merged**